setup.py

```diff
@@ -570,7 +573,14 @@ def get_ext_modules() -> List[Extension]:
             # Install the prebuilt library for custom ops used in llama.
             BuiltFile(
                 "examples/models/llama2/custom_ops/libcustom_ops_aot_lib.*",
-                "executorch/examples/models/llama2/custom_ops",
+                "executorch",
+            )
+        )
+        ext_modules.append(
+            # Install the prebuilt library for quantized ops required by custom ops.
+            BuiltFile(
+                "kernels/quantized/libquantized_ops_aot_lib.*",
+                "executorch",
```

Comment:
The second arg to BuiltFile is the name of the destination file or directory. If the intent is to install to a directory, then the destination should end in a `/` character. But please see if "executorch/data/lib/" works: that would address my comment above about not installing these in the root of the pip package, though you'd also need to update the RPATH. Or you could try setting the dest to `executorch/extension/pybindings/` so that the libs end up in the same directory as _portable_lib.so.

Reply:
Your script works! Yes the trailing `/` makes a difference